app-emulation/rkt/rkt-9999.ebuild

```diff
@@ -14,16 +14,16 @@ CROS_WORKON_PROJECT="coreos/rkt"
 CROS_WORKON_LOCALNAME="rkt"
 CROS_WORKON_REPO="git://github.com"
 
+
 if [[ "${PV}" == "9999" ]]; then
 	KEYWORDS="~amd64"
-	PXE_VERSION="738.1.0"
-
-elif [[ "${PV}" == "0.7.0" ]]; then
+elif [[ "${PV}" == "0.8.1" ]]; then
```

Comment:
We can probably just switch this to an else and add a `# 0.8.1` comment which is our more standard pattern.